src/open_company_web/components/su_snapshot_preview.cljs

```diff
@@ -137,9 +138,9 @@
           su-data      (stakeholder-update-data data)
           card-width   (responsive/calc-card-width 1)
           ww           (.-clientWidth (sel1 js/document :body))
-          total-width  (if (> ww 413) (str (min ww (+ card-width 100)) "px") "auto")
+          total-width  (if (> ww 413) (str (min ww (+ card-width 140)) "px") "auto")
```

Comment:
What's happening here? Some comments would be good. Also a little odd to have 2 constants like 413 and 140 w/o any defs.